### DISCUSSION TITLE
Aiuto per l'installazione

### DISCUSSION THREAD

**Discussion by cristianrussoo** [iisgiua/giuaschool#623] [Richieste di aiuto]
Salve,
vorrei provare il software in modo gratuito e sto cercando in quale hosting e possibilmente un tutorial su come importare i file all'interno dell'hosting, per il resto l'interno del software me lo studio un po' da me... è possibile avere un piccolo tutorial?

**Comment by ghost** [comment]
Buongiorno, la guida completa per l'installazione del Registro Elettronico, la trova qui: [installazione](https://iisgiua.github.io/giuaschool-docs/install-zip.html).
Cordialmente, spero di esserle stata d'aiuto.

**Comment by trinko** [comment]
Se si tratta solo di provare l'applicazione, consiglio di utilizzare la demo online:
vedi [messaggio](https://github.com/iisgiua/giuaschool/discussions/586)

**Comment by luigirotondi** [comment]
Salve, se ha bisogno di aiuto, può contattare anche registroelettronicogiuaschool@gmail.com in modo da aiutarti gratuitamente.
